content/acra/security-controls/searchable-encryption/_index.md

```diff
@@ -89,6 +93,68 @@ Such functions are very sensitive to the input and will return completely differ
 with the smallest change of input data (column value in our case).
 {{< /hint >}}
 
+## Searchable encryption with like
+
+[Acra Enterprise Edition](/acra/enterprise-edition/) also provide the ability of searchable encryption with `like`/`ilike` queries over encrypted fields.
+
+By specifying `searchable_prefix: <number>` config property under the column should be used in dynamic search, AcraServer/AcraTranslator will generate and store additional search hashes.
```

Comment:
As I remember, AcraTranslator doesn't support it now. And it doesn't use encryptor's config with `searchable_prefix` parameter